### PR TITLE
feat(cli): Add option to format dates in output files

### PR DESCRIPTION
**Pull request by wollanup**

### CODE REVIEW

**Comment by wollanup** [comment]
See https://github.com/invoice-x/invoice2data/issues/209

**Comment by m3nu** [comment]
Thanks. Will test this tomorrow.

**Comment by wollanup** [comment]
> Thanks. Will test this tomorrow.

Well, I forgot something, I'll fix it before tomorrow...

File: src/invoice2data/output/to_csv.py [hunk 1]
```diff
@@ -48,7 +50,7 @@ def write_to_file(data, path):
             csv_items = []
             for k, v in line.items():
                 # first_row.append(k)
-                if k == 'date':
-                    v = v.strftime('%d/%m/%Y')
+                if k.startswith('date') or k.endswith('date'):
```
Comment (m3nu) [inline src/invoice2data/output/to_csv.py]: `if 'date' in k:`

File: src/invoice2data/main.py [hunk 2]
```diff
@@ -117,6 +117,13 @@ def create_parser():
         help='Choose output format. Default: none',
     )
 
+    parser.add_argument(
+        '--output-date-format',
+        dest='output_date_format',
+        default='%d/%m/%Y',
+        help='Choose output date format. Default: %d/%m/%Y',
```
Comment (m3nu) [inline src/invoice2data/main.py]: Would suggest to use ISO 8601 (2019-02-22), even if it breaks stuff for some people.

File: src/invoice2data/output/to_csv.py (hunk 1 above)
Comment (wollanup) [reply src/invoice2data/output/to_csv.py]: Are you suggering to ignore `startswith()` and `endswith()` and only apply format for `date` key ?

File: src/invoice2data/main.py (hunk 2 above)
Comment (wollanup) [reply src/invoice2data/main.py]: OK!

**Comment by m3nu** [comment]
Merged. Sorry for the delay.

**Pull request merged**